src/test.rs

```diff
@@ -450,6 +450,55 @@ pub async fn ca_configured_roas(ca: &CaHandle) -> ConfiguredRoas {
     }
 }
 
+// short hand to expect ROA configurations in a CA
+pub async fn expect_configured_roas(ca: &CaHandle, expected: &[RoaConfiguration]) {
+    let configured_roas = ca_configured_roas(ca).await.unpack();
+    assert_eq!(configured_roas.len(), expected.len());
+
+    // Copy the expected configs, but convert them to an explicit max length because
+    // Krill always stores configs that way to avoid duplicate equivalent entries.
+    let expected: Vec<_> = expected
+        .iter()
+        .map(|entry| entry.clone().into_explicit_max_length())
+        .collect();
+
+    for configuration in configured_roas.iter().map(|configured| configured.roa_configuration()) {
+        if !expected.contains(configuration) {
+            use std::fmt::Write;
+
+            let mut expected_str = String::new();
```

Comment:
This seems like a lot of effort to do a join of a collection by commas into a string, isn't there a one-liner for this?

Reply:
yeah.. fixed it using a join call instead.